apps/search/test/unit/search_test.js

```diff
@@ -83,7 +90,7 @@ suite('search/search', function() {
       Search.init();
       MockNavigatormozApps.mTriggerLastRequestSuccess();
       assert.equal(MockNavigatormozApps.mLastConnectionKeyword,
-        'search-results');
+        'app-metrics');
```

Comment:
instead of removing the check for `search-results`, should you check for both `search-results` and `app-metrics` ?